sigstore-java/src/test/java/dev/sigstore/trustroot/SigstoreTrustedRootTest.java

```diff
@@ -117,4 +132,155 @@ void from_prod() throws Exception {
         "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEiPSlFi0CmFTfEjCUqF9HuCEcYXNKAaYalIJmBZ8yyezPjTqhxrKBpMnaocVtLJBI1eM3uXnQzQGAJdJ4gs9Fyw==",
         Base64.toBase64String(PublicKey.toJavaPublicKey(currCTLog.getPublicKey()).getEncoded()));
   }
+
+  @Test
+  public void getTlog_isPresent() {
+    var key = Base64.decode("wNI9atQGlz+VWfO6LRygH4QUfY/8W4RFwiT5i5WRgB0=");
+
+    // exactly at start time
+    assertIsPresent(
+        trustRoot.getTlog(key, ZonedDateTime.parse("2021-01-12T11:53:27.000Z").toInstant()));
```

Comment:
Have you considered factoring out `assertGetTLogExists` / `assertGetTLogMissing` so the assertion failures could include input data rather than "expected true got false"?

Reply:
I think the failure is pretty obvious from the data/test-line-number? In particular, someone would need to inspect the full trust root json and then decide why the test is failing.

It seems to me that changes would make the tests a bit more complicated that it needs to be? But maybe I'm not seeing this the same way you are?

Reply:
okay, I think coming back to this, I see what you're saying, lemme try it out